gcs/CHANGES.md

```diff
@@ -122,6 +122,8 @@
 
 1.  Upgrade java-storage to 2.22.3
 
+1.  Upgrade java-storage to 2.25.0
```

Comment:
should we just update the existing entry since otherwise there will be two upgrade messages in the release notes?

Reply:
sure